### DISCUSSION TITLE
请问我应该如何使用struct_pb，将protobuf数据序列化成相应结构体数据

### DISCUSSION THREAD

**Discussion by lnylgk** [alibaba/yalantinglibs#600] [Q&A]
### Search before asking

- [x] I searched the [issues](https://github.com/alibaba/yalantinglibs/issues) and found no similar issues.

### What happened + What you expected to happen
请问我应该如何使用struct_pb，将protobuf数据序列化成相应结构体数据

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

**Comment by lnylgk** [comment]
已解决

**Comment by poor-circle** [comment]
See document https://alibaba.github.io/yalantinglibs/en/struct_pb/struct_pb_quick_start.html
